xml/decoders/SoundTraxx_Tsu_SoundCar_Walthers.xml

```diff
@@ -2423,62 +2423,62 @@
       <!-- Consist function enable F13 to F28 (CV 245-246) -->
       <xi:include href="http://jmri.org/xml/decoders/soundtraxx/ECOCVconsistF13-28.xml"/>
       <!-- Flex-Map Function Mapping -->
-      <variable CV="16.1.259" item="Extended Function Map FX3 Effect" default="13" minOut="3">
```

Comment:
This and all similar changes need reverting.

Reply:
reverted